students/Lincoln_Zhang/Session4/trigrams.py

```diff
@@ -5,23 +5,35 @@
 #
 #
 #
-import random
+import random,sys
 
 from itertools import islice
 
-words = "I wish I may I wish I might".split()
+# words = "I wish I may I wish I might".split()
 
-for w in zip(words,islice(words,1,None),islice(words,2,None))
-    print(w)
+# for w in zip(words,islice(words,1,None),islice(words,2,None)):
+#     print(w)
 
 
 def read_file(fname):
```

Comment:
you tests for this function could make sure punctuation  was handles properly, etc.